<?xml version="1.0"?>
<launch>
	<include file="$(find nav_bgu)/launch/armadillo.launch">

          <arg name="have_map_file" value="true"/>
          <arg name="map_file" value="$(find nav_bgu)/maps/temp/second_floor_final_edited.yaml"/>

          <arg name="lidar" value="true"/>
          <arg name="move_base" value="true"/> 
          <arg name="kinect2" value="true"/>
          <arg name="amcl" value="true"/> 
          <arg name="joystick" value="true"/>
	  <arg name="moveit" value="true" />
        </include>

	<arg name="marker_size" default="4.4" />
	<arg name="max_new_marker_error" default="0.08" />
	<arg name="max_track_error" default="0.2" />
	<arg name="cam_image_topic" default="/kinect2/qhd/image_color" />
	<arg name="cam_info_topic" default="/kinect2/qhd/camera_info" />	
	<arg name="output_frame" default="kinect2_link" />

	<node name="ar_track_alvar" pkg="ar_track_alvar" type="individualMarkersNoKinect" respawn="false" output="screen" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic) $(arg cam_info_topic) $(arg output_frame)" />
</launch>
